app/processing/process-payment-request.js

```diff
@@ -15,6 +15,10 @@ const config = require('../config/processing')
 const processPaymentRequest = async (scheduledPaymentRequest) => {
   const { scheduleId, paymentRequest } = scheduledPaymentRequest
 
+  if (await hasAutoHold(paymentRequest)) {
+    return
+  }

```

Comment:
Concern with this approach is we might be blocking processing entirely.

Service processes 500 at a time with all non-processable excluded during the original SQL statement.

If the next 500 all have auto hold, then all will be filtered out.  Then they'll be picked up again on next attempt and you've got an endless loop of never processing anything